### ISSUE TITLE
"Ghost" user in issue list points 404 page, but should not be clickable

### ISSUE THREAD

**Issue by tklein23** [go-gitea/gitea#6270]
* When a user creates an issue, then the name in the issue list is clickable and points to the profile (which is correct)
* When the user is deleted, then the name changes to "Ghost", but is still clickable and sends to 404 page (which is not correct)

The incorrect line in the template is:
* https://github.com/go-gitea/gitea/blob/01bd1fcd33a7af55bfbba10f391b22a631ac592c/templates/repo/issue/list.tmpl#L223

In other places ghost links are handled correctly:
* https://github.com/go-gitea/gitea/blob/01bd1fcd33a7af55bfbba10f391b22a631ac592c/templates/repo/issue/view_title.tmpl#L39-L43

I was able to reproduce the bug in:
* https://try.gitea.io/
* https://github.com/Freeyourgadget/Gadgetbridge/issues

- Can you reproduce the bug at https://try.gitea.io:
  - [x] Yes
  - [ ] No
  - [ ] Not relevant

**Comment by tklein23** [comment]
The example on try.gitea.io (issue #2 in the list) where Ghost should not be clickable:
* https://try.gitea.io/pippo/fsafsdfsd/issues

**Comment by lunny** [comment]
@tklein23  could you send a PR to fix that?

**Comment by tklein23** [comment]
tl;dr: Sorry, I can't do that.

A CR would require additional translations for issues.closed_by and issues.merged_by in all supported languages.  IMO this is the wrong approach, because this increases the cost for adding new and supporting the existing languages. 

Someone with more knowledge about the template language should probably see if it's possible to fix the macro and avoid having a "_fake" variant of each issue event.

Maybe I can do that once I'm more familiar with the gitea codebase, but for sure I cannot do it right now.

**Comment by adelowo** [comment]
You don't have to do all of the translations. Only the English one is needed, others will be added as needed on https://crowdin.com .

**Comment by oscarlofwenhamn** [comment]
I think I've fixed the issue in the above commit, ~~but I'm not sure how to properly make a PR as I have another one pending, and committing seems to add it to that PR, sorry.~~
